src/block.js

```diff
@@ -1,12 +1,42 @@
 /**
  * External dependencies
  */
+import { getColorClassName } from '@wordpress/block-editor';
```

Comment:
This is out of the scope of your level, but `@wordpress/block-editor` shouldn't be called on the frontend. `block-editor` is a heavy package, that would severely hinder performance on the frontend.

Given that you're only using `getColorClassName`, can you just copy the function in an util file. Make sure to import `kebabCase` like this.

```
npm install lodash.kebabcase
```

```js
import kebabCase from 'lodash.kebabcase'

export function getColorClassName( colorContextName, colorSlug ) {
	if ( ! colorContextName || ! colorSlug ) {
		return undefined;
	}

	return `has-${ kebabCase( colorSlug ) }-${ colorContextName }`;
}
```

Reply:
Thanks for pointing this out. I just adjusted the code accordingly and updated this PR.